org.nickvision.tagger.json

```
@@ -0,0 +1,104 @@
+        {
+            "name": "jsoncpp",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.9.5.tar.gz",
+                    "sha256": "f409856e5920c18d0c2fb85276e24ee607d2a09b5e7d5f0a371368903c275da2"
+                }
+            ]
+        },
+        {
+            "name": "libcurl",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://curl.se/download/curl-7.83.1.tar.gz",
+                    "sha256": "93fb2cd4b880656b4e8589c912a9fd092750166d555166370247f09d18f5d0c0"
+                }
+            ]
+        },
+        {
+            "name": "curlpp",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/jpbarrette/curlpp/archive/refs/tags/v0.8.1.tar.gz",
+                    "sha256": "97e3819bdcffc3e4047b6ac57ca14e04af85380bd93afe314bee9dd5c7f46a0a"
+                }
+            ]
+        },
+        {
+            "name": "taglib",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://taglib.github.io/releases/taglib-1.12.tar.gz",
+                    "sha256": "7fccd07669a523b07a15bd24c8da1bbb92206cb19e9366c3692af3d79253b703"
+                }
+            ]
+        },
+        {
+            "name": "neon",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--with-libproxy=no",
+                "--with-gssapi=no"
+            ],
+
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://http.debian.net/debian/pool/main/n/neon27/neon27_0.30.2.orig.tar.gz",
+                    "sha256": "db0bd8cdec329b48f53a6f00199c92d5ba40b0f015b153718d1b15d3d967fbca"
+                }
+            ]
+        },
+        {
+            "name": "libmusicbrainz",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/metabrainz/libmusicbrainz/releases/download/release-5.1.0/libmusicbrainz-5.1.0.tar.gz",
+                    "sha256": "6749259e89bbb273f3f5ad7acdffb7c47a2cf8fcaeab4c4695484cef5f4c6b46"
+                }
+            ]
+        },
+        {
+            "name": "org.nickvision.tagger",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/nlogozzo/NickvisionTagger.git#tag=2022.6.1",
+                    "branch": "master"
```

you need a revision is definitely NOT a branch.

```suggestion
                    "tag": "2022.6.1"
```

based on above